pipeline/metadata/6.1.yaml

```diff
@@ -475,6 +475,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rbd
+        - name: "RGW bucket notifications testing on Multisite"
```

Comment:
@hmaheswa, could you please add this suite under stage-5, stage-6 alreday have 5 suites

Reply:
@anrao19 , moved the suites to stage-5 in 6.0 and 6.1 pipeline metadata